docs/faq/cr_by_components.md

```diff
@@ -1,61 +1,50 @@
-# Missing NNCPs
+# Creating Smaller CRs
+
+## Question
 
 The kustomize command builds and results in the OpenStack control
 plane definitions and its dependent Custom Resources (CR).
 ```bash
 kustomize build architecture/examples/va/hci > control-plane.yaml
 ```
-The `control-plane.yaml` file contains CRs for both `NMState` and
-`NodeNetworkConfigurationPolicy` (NNCP). When  `oc apply -f
-control-plane.yaml` is read, OpenShift will try to create the NNCPs
-while `NMState` Custom Resource Definitions (CRD) are still installing
-and produce a message noting that the resource mappings are not found:
-```
-nmstate.nmstate.io/nmstate created
-[resource mapping not found for name:
-"ostest-master-0" namespace: "openstack" from "control-plane.yaml":
-no matches for kind "NodeNetworkConfigurationPolicy" in version "nmstate.io/v1"
-ensure CRDs are installed first,
-resource mapping not found for name: "ostest-master-1" namespace: "openstack"
-from "control-plane.yaml": no matches for kind "NodeNetworkConfigurationPolicy"
-in version "nmstate.io/v1"
-```
-Retrying `oc apply -f contol-plane.yaml` a few seconds later is likely to
-resolve the problem.
+The `control-plane.yaml` file contains CRs for the
+`NodeNetworkConfigurationPolicy` (NNCP), the
```

Comment:
For avoiding OSPRH-3743 we need to split NNCP and ensure those are ready ready before moving forward with control-plane setup.

Reply:
I thought the issue was NNCP being created before NMState and MetalLB were ready and not from Networking CRs racing with NNCP. At least I couldn't reproduce the problem using the current patch. I could split NNCP to a separate sub directory though in a follow up.

Reply:
I think addressing NNCP split-out in a follow-up makes sense